docs/guide/best_practices.md

```diff
@@ -151,3 +151,12 @@ ezpublish:
 This setting **does not change Symfony behavior** regarding redirection after login. If set, it will only substitute the value set for `default_target_path`. It is therefore still possible to specify a custom target path using a dedicated form parameter.
 
 **Order of precedence is not modified.**
+
+### Copy subtree limit
+
+Copying large subtrees can cause performance issues, so you can limit the number of Content items
+that can be copied at once using `ezsystems.platformui.application_config.copy_subtree.limit`
+in `services.yml`.
```

Comment:
From perspective of developer using EZP, this parameter usually will be overwritten in `/app/config/parameters.yml` rather than in `services.yml` 😉